payslip.sh

```diff
@@ -183,6 +183,10 @@ TOKEN="$(wget ${VERBOSE} -O - \
   'https://fress2.adp.com/core/coreControl.asp?ProductType=0' \
   | grep sessionToken | cut -d "'" -f2)"
 
+if [ ${#TOKEN} = 0 ]; then
+   fail 'sessionToken not found in cookies.txt'
```

Comment:
Technically, this error message is wrong -- the session token is never in cookies it is in the response body of coreControl.asp

Reply:
Updated error message. Directing to look for actual values in cookies.txt. The missing values are a symptom I see when I test authentication errors.

Reply:
I'm not sure directing to cookies.txt is fair, as that is akin to asking a user to look in a private variable of the code -- cookies.txt is internal, and deleted after program execution.  We should really be using a trap to ensure that cookies.txt is always deleted, even in the event of failure, as it is a security leak to leave it lying around.

Reply:
I'll change the message to remove the reference to cookies.txt